src/domains/ecommerce-common/03_cosmosdb.tf

```diff
@@ -141,3 +141,56 @@ module "cosmosdb_ecommerce_collections" {
   shard_key   = each.value.shard_key
   lock_enable = var.env_short == "d" ? false : true
 }
+
+# -----------------------------------------------
+# Alerts
+# -----------------------------------------------
+
+resource "azurerm_monitor_metric_alert" "cosmos_db_normalized_ru_exceeded" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "[${var.domain != null ? "${var.domain} | " : ""}${module.cosmosdb_account_mongodb.name}] Normalized RU/s Exceeded"
+  resource_group_name = azurerm_resource_group.cosmosdb_ecommerce_rg.name
+  scopes              = [module.cosmosdb_account_mongodb.id]
+  description         = "A collection Normalized RU/s exceed provisioned throughput, and it's raising latency. Please, consider to increase RU."
+  severity            = 0
+  window_size         = "PT5M"
+  frequency           = "PT5M"
+  auto_mitigate       = false
+
+
+  # Metric info
+  # https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdocumentdbdatabaseaccounts
+  criteria {
+    metric_namespace       = "Microsoft.DocumentDB/databaseAccounts"
+    metric_name            = "NormalizedRUConsumpion"
+    aggregation            = "Maximum"
+    operator               = "GreaterThan"
+    threshold              = "90"
```

Comment:
@infantesimone 90% is very high as threshold maybe we don’t have time to intervene and we could have a disservice. Couldn’t we decide for the first period to lower the threshold to 80%? 🤔